### DISCUSSION TITLE
node_modules get included when searching files

### DISCUSSION THREAD

**Discussion by oscklm** [LazyVim/LazyVim#973] [Q&A] [answered]
I want to exclude node_modules folder from file search, is there a way to achieve this?

**Here is a GIF showing what i get when using <leader>ff**
![GIF](https://media.cleanshot.cloud/media/64988/WnZxludAu4hL629gFCwT7thnuF3c8xufmTupn1eT.gif?Expires=1687648910&Signature=eaYZdW~06mDHSm4yBdYAdLaaPq5cEpRljhg7ZtAfC~jOmQZiZ8RqULUFIbEX51VUWVBAUqz0Zyv5V8YV6ZBo5wAz6xR~qcko-SQ0apshBvCUHmGEn7g6LZA8ye8qLAhk~WWqfWVlaOtOj42S90fU0Wf2UZ10z8NZje5sdHyV6jMsX1oytY8NFlCeYIC1tOMsUo3N6nTHZM~PSgLH374eAoIdAUJ0N9UdcRSblt64pvBytEWUwjzSunQhpBZhqNcol7JUV26LjC5X79wEudwUcBlMYC0lY~gPYbNUiWxJNjqx9QuDl-k1ubgi-R7UVamQbtFZnCZpZpSO4AqyWzY~Dg__&Key-Pair-Id=K269JMAT9ZF4GZ)

Thanks in advance for any help!

**Comment by dpetka2001** [comment, marked as answer]
This [issue](https://github.com/nvim-telescope/telescope.nvim/issues/2471) on telescope repo should help you out. If you are in a git repo, if you add `node_modules` to `.gitignore` then telescope won't search in that directory anymore.

**Reply by oscklm** [reply]
Thanks a lot. It was in fact the project not being initialised to git. That fixed the issue.
